CHANGELOG.md

```diff
@@ -32,7 +40,28 @@ Date: 2023-04-28
 * :recycle: ⚠️ The `master` branch has been renamed to `main`
 * :recycle: The hadron sampler makes now use of the C++17 standard
 
-[Link to diff from previous version](https://github.com/smash-transport/smash/compare/SMASH-hadron-sampler-1.1...SMASH-hadron-sampler-3.0)
+[Link to diff from previous version](https://github.com/smash-transport/smash-hadron-sampler/compare/SMASH-hadron-sampler-1.1...SMASH-hadron-sampler-3.0)
 
 ## SMASH-hadron-sampler-1.1
 Date: 2022-05-17
+
+
+## Release checklist
+
+* Create a release branch from `develop` and switch to it
+* Adapt the _CMakeLists.txt_ file:
+  - bump the hard coded version macro (`VERSION` argument of `project(...)` command)
+  - comment out `set(SAMPLER_VERSION "${SAMPLER_VERSION}-next")` with a `#` to mark the codebase as clean
+* Adjust the _CHANGELOG.md_ file:
+  - add `## SMASH-hadron-sampler-X.Y.Z` below the `## Unreleased` line (leaving one empty line in-between). Prepare a `Date:` line below the release candidate section, which will be completed on the day of the public release, as well as the _Link to diff from previous version_ at the end of the upcoming release block (this will become a meaningful link after the public release)
+* If the new release is due to a new SMASH release, bump the mentioned SMASH version in the _README.md_ file section `Prerequisites`
+* Close the release branch in the git-flow sense:
+  - merge it into the `main` branch
+  - switch to `main` and tag the last commit
+  - switch to `develop` and merge the release back into it
```

Comment:
I would add here another last item to say to remove the release branch (if pushed).